test/algorithms/ground_state_solvers/test_adapt_vqe.py

```diff
@@ -108,7 +109,9 @@ def test_delta(self):
             quantum_instance=QuantumInstance(BasicAer.get_backend("statevector_simulator"))
         )
         delta1 = 0.01
-        calc = AdaptVQE(self.qubit_converter, solver, delta=delta1)
+        with warnings.catch_warnings(record=True):
+            warnings.simplefilter("always")
```

Comment:
Why is speciifcally this one needed?

Reply:
Because it was showing a deprecation warning from AdaptVQE, unnecessary in this unit test. I prefer doing this than adding another filter.

Reply:
Ah so you just found a warning unrelated to this PR which you are now filtering. Got it, thanks :+1: